build/pkgs/polymake/spkg-install.in

```diff
@@ -21,6 +21,10 @@ if [ -x $SAGE_LOCAL/bin/lrs1 ]; then
     more_configure_options="$more_configure_options --with-lrs=$SAGE_LOCAL"
 fi
 
+# Put these includes on the front, to avoid shadowing by installed headers
+# from a previous version. (polymake 4.8 puts these includes to the end.)
+export CXXFLAGS="-I$(pwd)/include/core-wrappers -I$(pwd)/include/core $CXXFLAGS"
```

Comment:
How did you find out, that you need to do this?

Reply:
There was an error when I tried the 4.8 upgrade, and I stared long enough at the command lines...

Reply:
Ok, there really is no reference to this solution.